akka-docs/rst/scala/code/docs/stream/cookbook/RecipeSimpleDrop.scala

```diff
@@ -15,11 +15,11 @@ class RecipeSimpleDrop extends RecipeSpec {
 
       //#simple-drop
       val droppyStream: Flow[Message, Message, NotUsed] =
-        Flow[Message].conflate(seed = identity)((lastMessage, newMessage) => newMessage)
+        Flow[Message].conflateWithSeed(seed = identity)((lastMessage, newMessage) => newMessage)
       //#simple-drop
       val latch = TestLatch(2)
       val realDroppyStream =
-        Flow[Message].conflate(seed = identity)((lastMessage, newMessage) => { latch.countDown(); newMessage })
+        Flow[Message].conflateWithSeed(seed = identity)((lastMessage, newMessage) => { latch.countDown(); newMessage })
```

Comment:
The stages-overview should also mention it, but I can add that in the stages overview update ticket.